manifests/c/Chatterino/Chatterino/2.3.2/Chatterino.Chatterino.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Chatterino.Chatterino
+PackageVersion: 2.3.2
+PackageLocale: en-US
+Publisher: Chatterino
```

Comment:
```suggestion
Publisher: Daniel Schnabel
```

Reply:
https://github.com/microsoft/winget-pkgs/pull/16022#issuecomment-860028800

Reply:
cc @fourtf @DRSchlaubi `Publisher` must match the one in Control Panel; otherwise, please update your software to change it to something else.